inc/options/class-wpseo-option-wpseo.php

```diff
@@ -262,6 +264,8 @@ protected function validate_option( $dirty, $clean, $old ) {
 					}
 					break;
 				case 'indexing_reason':
+				case 'category_base_url':
+				case 'tag_base_url':
```

Comment:
Why these values are now in a validation that uses `sanitize_text_field()`? I thought Herre's feedback was:
- either use the same validation used for `permalink_structure`, which is `sanitize_option()`
- or check if all these 3 values really need validation and if not (because they may be already stored by WP and already sanitized) use the simplest validation that only checks if they're set

Reply:
They are moved to `sanitize_option()`